src/validate-addresses/types/public.ts

```diff
@@ -1,5 +1,4 @@
-import { Country, ValidationMessageType } from "../../enums";
-import { ValidationMessageCode } from "../../constants";
+import { ValidationMessageCode, ValidationMessageType } from "../../constants";
```

Comment:
A question in general about the way this is done. It limits the type Address to be the public interface for the input but also the same address is used in the response. Should these be 2 different addresses?

Not relevant to this pr, but something I would want to know.